### PR TITLE
Update to Hoomd V3

### PR DESCRIPTION
**Pull request by chrisjonesBSU**
This PR updates everything to run on hoomd v3. 

There are a couple issues that need to be resolved else where.
One is a PR I'll make to hoomd, and another is fixing how hoomd is used/imported in cmeutils

### CODE REVIEW

File: polybinder/simulate.py [hunk 1]
```diff
@@ -364,178 +343,181 @@ def anneal(
...
-                group=_all,
-                phase=0,
-                dynamic=["momentum"],
-                overwrite=False,
+        init_x = init_snap.configuration.box[0]
+        init_y = init_snap.configuration.box[1]
+        init_z = init_snap.configuration.box[2]
+        if isinstance(self.nlist, hoomd.md.nlist.Tree):
+            forcefields[0].nlist = self.nlist(buffer=0.4)
+        forcefields[0].nlist.exclusions = ["bond", "1-3", "1-4"]
```
Comment (jennyfothergill) [inline polybinder/simulate.py:421]: You may not want this hardcoded. GAFF has 1-4 exclusions, but I'm not sure about other forcefields. `create_hoomd_forcefield` should do handle setting the exclusions automatically unless the 1-4 scaling factor is 0, and then it'll need to be set manually.

**Review by jennyfothergill — APPROVED** [review]
LGTM!

File: polybinder/simulate.py (hunk 1 above)
Comment (chrisjonesBSU) [reply polybinder/simulate.py:421]: Good point, do you know if I need to reset the exclusions when changing the nlist type, like changing it to a tree?

Comment (jennyfothergill) [reply polybinder/simulate.py:421]: I think so because you are creating a completely new object, so it would have no knowledge of the previous exclusions.

**Review by jennyfothergill — COMMENTED** [review]
I think this should work because the exclusions are just a list, but you might want to verify that hoomd won't complain if you try to attach exclusions from an existing neighborlist to a new one.

**Comment by chrisjonesBSU** [comment]
> 

So far it's passing the unit test that does change to nlist to a Tree. Maybe it would be safer to copy to contest of the original nlist.exclusions to a new list and use that?

**Pull request merged**